payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt

```diff
@@ -59,6 +62,20 @@ class StripePaymentProcessor : PaymentProcessor {
                 ProfileInfo(Customer.create(customerParams).id)
             }
 
+    override fun getPaymentProfile(userEmail: String): Either<PaymentError, ProfileInfo> {
+            val customerParams = mapOf(
+                    "limit" to "1",
+                    "email" to userEmail)
+            val customerList = Customer.list(customerParams)
+            if (customerList.data.isEmpty()) {
+                 return Either.left(NotFoundError("Could not find a payment profile for user  $userEmail"))
+            } else if (customerList.data.size > 1){
+                 return Either.left(NotFoundError("Multiple profiles for user $userEmail found"))
```

Comment:
Is this allowed in Stripe (having multiple profiles with same email ) ?

Reply:
I am not sure. The Documentation does not mention if it must be unique.
https://stripe.com/docs/api#create_customer
So I added a check incase :)